lua/defaults.lua

```diff
@@ -9,6 +9,7 @@ lang_conf["markdown.pandoc"] = { "```", "```" }
 
 M.lang_conf = lang_conf
 
+M.virt_text = false
```

Comment:
Please rename here and everywhere to `virtual_text` in order to increase readability.

Reply:
We also need this in the documentation and an example in the README.

Reply:
The users will also need this in the documentation and the example in README.